config/keeperfx.cfg

```diff
@@ -13,8 +13,10 @@ FRONTEND_RES=640x480x32 640x480x32 640x480x32
 ; Original DK resolutions were FRONTEND_RES=320x200x8 320x200x8 640x480x8
 ; List of in-game resolutions. ALT+R will switch between them.
 INGAME_RES=640x480x32 1024x768x32
-;Original DK resolutions were INGAME_RES=320x200x8 640x400x8
+; Original DK resolutions were INGAME_RES=320x200x8 640x400x8
```

Comment:
Seems unrelated to that change :)

Reply:
You're right. While updating the file I just added the missing space to make it uniform with the other lines. Is not a functional change, just something snuck in.